main.js

```diff
@@ -0,0 +1,240 @@
+    technology: ['html', 'Bootstrap', 'Ruby'],
+    languages: ['html', 'Bootstrap', 'Ruby on rails'],
+    description: 'A daily selection of privately personalized reads;no accounts or sign-ups required. has been the industrys standard',
+    objectdescription: 'Lorem Ipsum is simply dummy text of the printing and type setting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita',
+    livelink: 'https://stevenmukama.github.io/Microverseportfolio/',
+    sourceLink: 'https://github.com/stevenmukama/Microverseportfolio',
+  },
+  {
+    title: 'Data Dashboard Healthcare',
+    popupdescription: 'Multi Post Stories',
+    description: 'A daily selection of privately personalized reads no accounts or sign-ups required. has been the industrys standard',
+    objectdescription: 'Lorem Ipsum is simply dummy text of the printing and type setting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita',
+    technology: ['html', 'Bootstrap', 'Ruby'],
+    languages: ['html', 'Bootstrap', 'Ruby on rails'],
+    // image: 'SVG/bgw2.svg',
+    livelink: 'https://stevenmukama.github.io/Microverseportfolio/',
+    sourceLink: 'https://github.com/stevenmukama/Microverseportfolio',
+  },
+  {
+    title: 'Website Protfolio',
+    popupdescription: 'Multi Post Stories',
+    description: 'A daily selection of privately personalized reads no accounts or sign-ups required. has been the industrys standard',
+    objectdescription: 'Lorem Ipsum is simply dummy text of the printing and type setting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita',
+    technology: ['html', 'Bootstrap', 'Ruby'],
+    languages: ['html', 'Bootstrap', 'Ruby on rails'],
+    // image: 'SVG/w6bgdt.png',
+    livelink: 'https://stevenmukama.github.io/Microverseportfolio/',
+    sourceLink: 'https://github.com/stevenmukama/Microverseportfolio',
+  },
```

Comment:
- [ ] There are seven cards with the button `See project` you should have seven pop-up windows. Please 🙏🏽 add the pop-up window for a better user experience.

![image](https://user-images.githubusercontent.com/58642949/202827367-1f95943c-2975-47f5-8607-26cb499c6cc0.png)